packages/pilot/src/formatters/error.js

```diff
@@ -0,0 +1,29 @@
+const getErrorMessage = ({ message, response }) => {
+  if (response && response.errors) {
+    const error = response.errors[0]
+    return error.message
+      ? error.message
+      : message
+  }
+
+  return message
+}
+
+const getErrorName = ({ name }) => name || 'unknown'
```

Comment:
Aqui poderia ser utilizado um propOr
```suggestion
const getErrorName = propOr('unknown', 'name')
```